support/notifications-client/client_test.go

```diff
@@ -0,0 +1,112 @@
+//
+// Copyright (c) 2017 Tencent
```

Comment:
We're in 2018.

Reply:
I will change this, I just follow the pre-existing style before.

Reply:
@drasko changed it.